Tasks/Common/webdeployment-common-v2/jsonvariablesubstitutionutility.ts

```diff
@@ -73,19 +73,19 @@ export function substituteJsonVariableV2(jsonObject, envObject) {
             if(resultNode.isEnd) {
                 switch(typeof(jsonObject[jsonChild])) {
                     case 'number':
-                    tl.debug('substituting value on key: ' + jsonChild + ' with (number) value: ' + resultNode.value);
+                        console.log(tl.loc('SubstitutingValueonKey' + jsonChild + 'SubstitutingWithNumberValue' + resultNode.value));
```

Comment:
tl.l.oc("SubstitutingValueonKeyWithNumber, "JsonChild", resultNode.value) should be the way to print statemets